cmd/openshift-controller-manager/main.go

```diff
@@ -63,10 +64,10 @@ func NewOpenShiftControllerManagerCommand(stopCh <-chan struct{}) *cobra.Command
 			os.Exit(1)
 		},
 	}
-	start := openshift_controller_manager.NewOpenShiftControllerManagerCommand("start", os.Stdout, os.Stderr, stopCh)
+	start := openshift_controller_manager.NewOpenShiftControllerManagerCommand("start", os.Stdout, os.Stderr, ctx)
 	cmd.AddCommand(start)
 
-	routeStart := route_controller_manager.NewRouteControllerManagerCommand("route-controller-manager-start", os.Stdout, os.Stderr, stopCh)
+	routeStart := route_controller_manager.NewRouteControllerManagerCommand("route-controller-manager-start", os.Stdout, os.Stderr, ctx.Done())
```

Comment:
I am not updating the route controllers code as this functionality should be removed soon and moved to https://github.com/openshift/route-controller-manager/pull/6